odd_collector_gcp/adapters/bigtable/mapper.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+from functools import reduce
+from operator import iconcat
+from typing import List
+
+from odd_models.models import (
+    DataEntity,
+    DataEntityType,
+    DataEntityGroup,
+    DataSet,
+    DataSetField,
+    DataSetFieldType,
+    Type,
+)
+from oddrn_generator import BigTableGenerator
+
+from odd_collector_gcp.adapters.bigtable.models import (
+    BigTableInstance,
+    BigTableTable,
+    BigTableColumn,
+)
+
+
+TYPE_MAPPING = {
+    str: Type.TYPE_STRING,
+    float: Type.TYPE_NUMBER,
+    int: Type.TYPE_INTEGER,
+    bool: Type.TYPE_BOOLEAN,
+    datetime.datetime: Type.TYPE_DATETIME,
+    datetime.time: Type.TYPE_TIME,
+    dict: Type.TYPE_STRUCT,
+    bytes: Type.TYPE_BINARY,
+    list: Type.TYPE_LIST,
+    tuple: Type.TYPE_UNION,
+    datetime.timedelta: Type.TYPE_DURATION,
+}
+
+
+class BigTableMapper:
+    def __init__(self, oddrn_generator: BigTableGenerator):
+        self.__oddrn_generator = oddrn_generator
+
+    def map_instances(self, instances: List[BigTableInstance]) -> List[DataEntity]:
+        return reduce(iconcat, [self.map_instance(i) for i in instances], [])
```

Comment:
Looks like you implement [lmapcat](https://funcy.readthedocs.io/en/stable/seqs.html?highlight=lmapcat#lmapcat) fn